dev.yml

```diff
@@ -6,8 +6,12 @@ up:
   - docker-compose  # Not working yet
   - custom:
       name: Install SASSC
-      met?: which sassc
+      met?: which sassc > /dev/null
```

Comment:
Maybe that should become a default in dad ?

Reply:
Yes!